src/libs/ReportUtils.js

```diff
@@ -154,6 +154,15 @@ function isAnnounceRoom(report) {
     return lodashGet(report, ['chatType'], '') === CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE;
 }
 
+/**
+ * Attempts to find a report in onyx with the provided list of participants
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function isIOUReport(report) {
+    return !!lodashGet(report, 'total');
```

Comment:
```suggestion
    return !lodashGet(report, 'total');
```

Reply:
Why don't we like `!!`? It turns it into a boolean!

Reply:
Oh i see the problem now, well I think `!!` is a kosher solution here 😄

Reply:
I think this would be the best.
```suggestion
    return _.has(report, 'total');
```

Reply:
Well, the function is to define that the report is IOU by getting the `total`. If the report supplied is `undefined` or `null` the resolved value from `lodashGet` is `undefined`. So `!!lodashGet(report, 'total')` it makes sense to me.

Or we can use the `Boolean` constructor.
```js
return Boolean(lodashGet(report, 'total'));
```

Reply:
In that case I would prefer this honestly:

`return report && _.has(report, 'total');`